packages/core/src/tools/create-app-tester.js

```diff
@@ -29,7 +29,7 @@ const createAppTester = (appRaw, { customStoreKey } = {}) => {
 
   const randomSeed = genId();
 
-  return (methodOrFunc, bundle) => {
+  const appTester = (methodOrFunc, bundle, clearZcacheBeforeUse = null) => {
```

Comment:
This refactor of the `createAppTester` used for testing integrations allows devs to test the caching part of their integration code. Two things:
- It uses an object `zcacheTestObj` saved on this function `appTester` as the cache local storage. So the production cache won't be used during testing.
- It uses argument `clearZcacheBeforeUse` to know when to clear the cache in the test case. So devs have a way to ensure an empty cache is used for separate test cases.

A typical example of how devs would test caching is done here https://github.com/zapier/zapier-platform/pull/646/files#diff-e507cf9820f17d4bfa0d153e57ca25f22daf1b0f0e7dd7a44c86deb9a3f44846